libs/sdk-ui-dashboard/src/model/store/filterContext/filterContextReducers.ts

```diff
@@ -411,6 +411,33 @@ export interface IUpdateConnectingAttributesOnFilterAddedPayload {
     connectingAttributes: IParentWithConnectingAttributes[];
 }
 
+export interface IChangeAttributeTitlePayload {
+    readonly filterLocalId: string;
+    readonly title?: string;
+}
+
+/**
+ * Changes the title for the filter given by its local identifier.
+ */
+const changeAttributeTitle: FilterContextReducer<PayloadAction<IChangeAttributeTitlePayload>> = (
+    state,
+    action,
+) => {
+    invariant(state.filterContextDefinition, "Attempt to edit uninitialized filter context");
+
+    const { filterLocalId, title } = action.payload;
+
+    const currentFilterIndex = state.filterContextDefinition.filters.findIndex(
```

Comment:
I would replace this with find

Reply:
Sorry, what do you refer to? The name of the variable or the `findIndex` method?

Reply:
Updated. Thank you!